From 6b227adac5dbabc958ef215f15637d3c4d21d759 Mon Sep 17 00:00:00 2001
From: Yan <yanxk.fnst@fujitsu.com>
Date: Sat, 7 Oct 2023 18:27:42 +0800
Subject: [PATCH] boa: modify conf file matching AGL system

set boa and boa_index working path to /usr/bin
error log and access log are re-directed to /var/log/
default index reset to homepage
cgi-bin folder to /usr/bin/cgi-bin

Upstream-Status: Pending
Signed-off-by: Yan <yanxk.fnst@fujitsu.com>
---
 boa.conf | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/boa.conf b/boa.conf
index e3775db..4da9983 100644
--- a/boa.conf
+++ b/boa.conf
@@ -59,7 +59,7 @@ Group nogroup
 # Set to /dev/null if you don't want errors logged.
 # If unset, defaults to /dev/stderr
 
-ErrorLog /var/log/boa/error_log
+ErrorLog /var/log/boa_error_log
 # Please NOTE: Sending the logs to a pipe ('|'), as shown below,
 #  is somewhat experimental and might fail under heavy load.
 # "Usual libc implementations of printf will stall the whole
@@ -71,7 +71,7 @@ ErrorLog /var/log/boa/error_log
 # Comment out or set to /dev/null (less effective) to disable 
 # Access logging.
 
-AccessLog /var/log/boa/access_log
+AccessLog /var/log/boa_access_log
 # Please NOTE: Sending the logs to a pipe ('|'), as shown below,
 #  is somewhat experimental and might fail under heavy load.
 # "Usual libc implementations of printf will stall the whole
@@ -108,7 +108,7 @@ AccessLog /var/log/boa/access_log
 # DocumentRoot: The root directory of the HTML documents.
 # Comment out to disable server non user files.
 
-DocumentRoot /var/www
+DocumentRoot /etc/boa/www
 
 # UserDir: The name of the directory which is appended onto a user's home
 # directory if a ~user request is recieved.
@@ -120,14 +120,14 @@ UserDir public_html
 # fly creation of directory indexes can be _slow_.
 # Comment out to always use DirectoryMaker
 
-DirectoryIndex index.html
+DirectoryIndex homepage.html
 
 # DirectoryMaker: Name of program used to create a directory listing.
 # Comment out to disable directory listings.  If both this and
 # DirectoryIndex are commented out, accessing a directory will give
 # an error (though accessing files in the directory are still ok).
 
-DirectoryMaker /usr/lib/boa/boa_indexer
+DirectoryMaker /usr/bin/boa_indexer
 
 # DirectoryCache: If DirectoryIndex doesn't exist, and DirectoryMaker
 # has been commented out, the the on-the-fly indexing of Boa can be used
@@ -190,5 +190,5 @@ Alias /doc /usr/doc
 # ScriptAlias: Maps a virtual path to a directory for serving scripts
 # Example: ScriptAlias /htbin/ /www/htbin/
 
-ScriptAlias /cgi-bin/ /usr/lib/cgi-bin/
+ScriptAlias /cgi-bin/ /usr/bin/cgi-bin/
 
-- 
2.25.1

